docs/guide/node_in_cloud.md

```diff
@@ -0,0 +1,152 @@
+# Deploy tfjs-node project on cloud platform
+
+This doc describes how to run a Node.js process with @tensorflow/tfjs-node package on cloud platforms.
+
+Starting from tfjs-node@1.2.4, running Node.js project on cloud platforms does not require extra configurations. This guide will use the [mnist-node](https://github.com/tensorflow/tfjs-examples/tree/master/mnist-node) example in @tensorflow/tfjs-examples repository, which trains MNIST with tfjs-node package on Heroku and GCloud. The details of Heroku’s Node.js Support can be found in [Heroku Node.js Support](https://devcenter.heroku.com/articles/nodejs-support) article. The details of Node.js on Google Cloud Platform can be found on the [Google Cloud page](https://cloud.google.com/nodejs/docs/).
```

Comment:
Starting from tfjs-node@1.2.4, running Node.js project on cloud platforms does not require additional configuration. This guide will show how to run the [mnist-node](https://github.com/tensorflow/tfjs-examples/tree/master/mnist-node) example in the @tensorflow/tfjs-examples repository on Heroku and GCloud. Heroku’s Node.js support is documented in this [article](https://devcenter.heroku.com/articles/nodejs-support). Running Node.js on Google Cloud Platform is documented [here](https://cloud.google.com/nodejs/docs/).

Reply:
Done.